src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task{
+    private String deadline;
+    public Deadline(String description, String deadline){
```

Comment:
```suggestion
    public Deadline(String description, String deadline) {
```

Similar to most of your methods, do remember to leave a space between the closing parenthesis and the opening curly brace.